transport_test.go

```diff
@@ -230,38 +230,38 @@ func TestGetTransportCallerName(t *testing.T) {
 	}
 }
 
-func TestGetTransportTraceEnabled(t *testing.T) {
-	s := newServer(t)
-	defer s.shutdown()
-	s.register("test", methods.traceEnabled())
-
-	tests := []struct {
-		benchmarking bool
-		traceEnabled byte
-	}{
-		{true, 0},
-		{false, 1},
-	}
-
-	opts := TransportOptions{
-		ServiceName: s.ch.ServiceName(),
-		HostPorts:   []string{s.hostPort()},
-	}
-
-	for _, tt := range tests {
-		opts.benchmarking = tt.benchmarking
-
-		ctx, cancel := tchannel.NewContext(time.Second)
-		defer cancel()
-
-		tchan, err := getTransport(opts, encoding.Raw)
-		require.NoError(t, err, "getTransport failed")
-		res, err := tchan.Call(ctx, &transport.Request{Method: "test"})
-		require.NoError(t, err, "transport.Call failed")
-
-		assert.Equal(t, tt.traceEnabled, res.Body[0], "TraceEnabled mismatch")
-	}
-}
+// func TestGetTransportTraceEnabled(t *testing.T) {
```

Comment:
Is there a better way to disable the test than by commenting it out? For example, something like a:

```
t.Skip("reason this is being skipping")
```